cmd/libs/go2idl/client-gen/testdata/apis/testgroup/v1/register.go

```diff
@@ -41,6 +42,7 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&v1.DeleteOptions{},
 		&unversioned.Status{},
 		&v1.ExportOptions{})
+	versionedwatch.AddToGroupVersion(scheme, SchemeGroupVersion)
```

Comment:
This is defining an API group, that is used by `examples/apiserver/apiserver.go` to serve API requests.  To serve api requests you need to have the WatchEvent API type defined now.